tests/timeseries/test_quantities.py

```diff
@@ -45,6 +48,38 @@ class Fz2(
     """Frequency quantity with broad exponent unit map."""
 
 
+_CtorType = Callable[[float], Quantity]
+
+_QUANTITY_SUBCLASSES = [
+    cls
+    for _, cls in inspect.getmembers(
+        _quantities,
+        lambda m: inspect.isclass(m) and issubclass(m, Quantity) and m is not Quantity,
+    )
+]
+# Make a simple sanity check, 7 is the current number of subclasses
+assert len(_QUANTITY_SUBCLASSES) >= 7
```

Comment:
you might want to define a variable like `NUM_SUBCLASSES=7` so that if new subclasses are added in the future we will only need to modify the variable. We have the comparison to number of subclasses twice, so also feel free to discard this comment if it is not worth the effort

Reply:
Thanks!